CONTRIBUTING.md

```diff
@@ -33,6 +33,59 @@ For example, see [mongo-go-driver tests configuration](https://github.com/Ferret
 `TEST` environment variable should have the value `mongo-go-driver`, or be empty.
 It defines what test configuration to run; empty value runs all configurations.
 
+### Running custom tests
```

Comment:
```suggestion
### Adding tests
```

We want to document how to add tests to Dance CI, not just how to run them with Dance (which does not have much value – they already know how to run their tests, and using Dance has no benefits for them)